src/Lykke.Service.MicrographCache.Services/HistoryService.cs

```diff
@@ -1,30 +1,44 @@
 ﻿using System;
-using System.Globalization;
-using System.Linq;
+using System.Collections.Generic;
 using System.Threading.Tasks;
 using Lykke.Service.MicrographCache.Core.Repositories;
 using Lykke.Service.MicrographCache.Core.Services;
-using Lykke.Service.MicrographCache.Services.Extensions;
-using Microsoft.Extensions.Caching.Distributed;
 
 namespace Lykke.Service.MicrographCache.Services
 {
     public class HistoryService : IHistoryService
     {
-        private readonly IDistributedCache _distributedCache;
         private readonly IFeedHoursHistoryRepository _feedHoursHistoryRepository;
-        private readonly TimeSpan _cacheExpiration;
+        private Dictionary<string, double[]> _cache;
```

Comment:
Please consider using ConcurrentDictionary to avoid "collection was modified" exceptions in case when data was requested at the "update cache" moment.